flatpak_builder_lint/checks/modules.py

```diff
@@ -2,35 +2,35 @@
 
 
 class ModuleCheck(Check):
-    type = "manifest"
+    type = "module"
```

Comment:
This is a reversal of your changes in commit b253e61 because the checker is called with a module not with a full manifest.

We could change this, but I found this is a nicer way to go at it.

Reply:
I could see it as having a type "manifest:module"